Updated ms.topic type (#2615)
MicrosoftDocs/msteams-docs: msteams-platform/bots/calls-and-meetings/debugging-local-testing-calling-meeting-bots.md

---

> title: How to develop calling and online meeting bots on your local PC

Reviewer: ```suggestion
title: Debug your calling and meeting bot locally
```
